Dev/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js

```diff
@@ -181,8 +183,75 @@ export class ClassMainScreen extends QcParentScreen {
                 })}
               />
             </View>
+            {

```

Comment:
How about students who haven't set their status to "working on it", do we have a default status as "assigned" for this? Also do we add a category for student with no assignment?

So the status state machine would be something like this:
student with no assignment: ==> status: No assignments 
Teacher assigns an assignment ==> status: Assigned
Student marks the assignment as working on it ===> status: Working on it
Student marks the assignment as need help: ==> status: Needs help 
-- note: how does the status gets cleared? Can teacher clear this status from their end? we should think about that user flow.
Student marks the assignment as ready: ==> status: Ready
Teacher evaluates the tasmee3 ===> status: No assignments

What do you think?